### DISCUSSION TITLE
How to add hyperlinks to cell text?

### DISCUSSION THREAD

**Discussion by cgreenland** [winsiderss/systeminformer#1454] [Questions / Answers] [answered]
Hello,

In relation to the project mentioned in [1373](https://github.com/winsiderss/systeminformer/issues/1373), we're trying to enhance the scan results column with links to multi-scanner reports. Does anyone know how we can easily make the text "x | y" (i.e. in the VirusTotal column) a clickable link within the OnlineChecks plugin? There are already hyperlinks in the About dialog, but we're not sure how to get hyperlinks in cell text. Thank you.

**Comment by dmex** [comment, marked as answer]
The treelist control only supports mouseover events for the rows not for individual cells. You would need mouseover events for the cell to implement a clickable link within the cell.

**Reply by cgreenland** [reply]
Thank you for the information. Where can I find more documentation of the treelist control used (we're new to Windows programming)? Are mouseover events for the cell something we could bolt on ourselves or is that non-trivial? We're trying to implement something similar to these links in Process Explorer (image [credit](https://github.com/winsiderss/systeminformer/issues/872#issuecomment-823093209)). 
![image](https://user-images.githubusercontent.com/57048220/199128791-732199c7-cfa9-45d1-8df5-1192def8004b.png)

Thank you again.
